packages/lib/src/components/NavBar/components/NavLinks.tsx

```diff
@@ -34,14 +35,16 @@ type Props = {
 export function NavLinks({ links, userRights }: Props): JSX.Element {
   const { t } = useTranslation()
 
-  const router = useRouter()
+  const pathname = window.location.pathname.includes('de')
+    ? window.location.pathname.replace('de', '')
+    : window.location.pathname
```

Comment:
We should use [`usePathname`](https://nextjs.org/docs/app/api-reference/functions/use-pathname) here.